<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="typeNameLine">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="playersTextBrowser">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="playersLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Players:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="infoTextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Add players and click &amp;quot;Start Game&amp;quot; when ready&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="typeLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type player name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startGamePushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>731</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gameTableGridLayout"/>
   </widget>
   <widget class="QPushButton" name="addPlayerPushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add player</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cardsSpinBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>43</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="cardsLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cards:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
